packages/aws-cdk-lib/scripts/minify-sources.sh

```diff
@@ -16,4 +16,12 @@
 scriptdir=$(cd $(dirname $0) && pwd)
 cd ${scriptdir}/..
 
-find . -name '*.js' ! -name '.eslintrc.js' ! -path '*node_modules*' | xargs npx esbuild --sourcemap --platform=node --format=cjs --minify-whitespace --minify-syntax --tsconfig=tsconfig.json --allow-overwrite --outdir=.
+find . -name '*.js' ! -name '.eslintrc.js' ! -path '*node_modules*' | xargs npx esbuild \
+  --sourcemap \
+  --platform=node \
+  --format=cjs \
+  --minify-whitespace \
+  --minify-syntax \
+  --tsconfig=tsconfig.json \
+  --allow-overwrite \
+  --outdir=.
```

Comment:
There's no actual change here, right? just formatting? Can you revert?

Reply:
oh wait, you've added `--minify-whitespace`. What impact does that have?

Reply:
It's formatting that Rico wanted. I can revert and separate it into a different PR, but I figured since its formatting it wouldn't hurt the integrity of this PR.

Reply:
This comment is sufficient documentation for me as to why the change is here.